### ISSUE TITLE
Allow component preview on editor.

### ISSUE THREAD

**Issue by johannbuscail** [measuredco/puck#364]
The editor currently lacks the capability for users to test dynamic components within it. 

Implementing a "Preview" mode, accessible through a checkbox (near the delete & duplicate btns), could significantly enhance usability. In this mode, users wouldn't be able to reposition the component but could interact with its functionalities. 

Such a feature would game changer the editor by enabling the creation and testing of dynamic, editable components directly within the editor.

For instance, users could adjust the settings of a "tabs" component via the sidebar and immediately experience its behavior in the editor, facilitated by this proposed checkbox.

**Comment by chrisvxd** [comment]
We did originally have "Alt" as a hotkey to enable direct interaction, but it was undocumented and got removed at some point.

Keen to explore this again.

**Comment by rv-bparise** [comment]
Any update on this functionality?

**Comment by chrisvxd** [comment]
Yes, this has been reintroduced in #763 via the `meta+i` hotkey and will be released in 0.18. It's also now controllable via the app state. Currently available in the 0.18 canary.
